rpc_jobs/rpc_aio.yml

```diff
@@ -335,6 +343,11 @@
                   holland.holland()
                 }} else if (env.STAGES.contains("Leapfrog Upgrade")) {{
                   deploy.upgrade_leapfrog(environment_vars: environment_vars)
+                  maas.deploy()
+                  maas.verify()
+                  tempest.tempest()
+                  kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
+                  holland.holland()
```

Comment:
We still add the Holland stage when it's a periodic job, so we can leave this until we do a wholesale remove of Holland from rpc-gating.